routes/ylist.js

```diff
@@ -19,9 +19,7 @@ router.get('/logout', (req, res)=>{
     let userName = dataFunc.searchForCookie(req.cookies['auth0']);
     if(userName){
         res.cookie('auth0', 0,{httpOnly: true});
-        if(dataFunc.sessionDelete(userName.userName)==1){
```

Comment:
is that really necessary to make in if statement and then send the number that the function will return. just return function value